src/library/basm.c

```diff
@@ -1271,7 +1276,7 @@ Macrodef *basm_resolve_macrodef(Basm *basm, String_View name)
     return NULL;
 }
 
-void basm_save_to_file_as_nasm_sysv_x86_64(Basm *basm, Syscall_Target target, const char *output_file_path)
+void basm_save_to_file_as_nasm_x86_64(Basm *basm, Syscall_Target target, const char *output_file_path)
```

Comment:
I'm not really sure about the name `Syscall_Target` here anymore. Essentially you're not only changing syscall numbers but also the ABI ...

What would make more sense is, to split this into two parts:
- System V ABI / Microsoft ABI (because you're not really making Windows Syscalls...)
- Specifying the OS for System V in a seperate parameter

See: https://en.wikipedia.org/wiki/X86_calling_conventions#x86-64_calling_conventions

I know this is me being a little pedantic with correctness, but: it'll make future porting efforts easier, as leaving this as is, is going to turn more and more into a mess.

Reply:
Thanks, I'm gonna look into that!